provers/celestia-prover/.env

```diff
@@ -0,0 +1,15 @@
+# SP1_PROVER={network|local|mock}
+SP1_PROVER=mock
+# Private key with the permission to use the network prover (optional if you use SP1_PROVER=mock)
+SP1_PRIVATE_KEY="PRIVATE_KEY"
+# Private key which the operator uses to sign the transactions in Eth Sepolia testnet
+PRIVATE_KEY="0x82bfcfadbf1712f6550d8d2c00a39f05b33ec78939d0167be2a737d691f33a6a"
+# URL of the Tendermint RPC node
+TENDERMINT_RPC_URL=http://localhost:5123
+# URL of the Ethereum RPC node
+# use https://ethereum-sepolia.publicnode.com/ for the Eth Sepolia testnet
+RPC_URL=http://localhost:8545
+# Address of the light client contract
+CONTRACT_ADDRESS="0x2854CFaC53FCaB6C95E28de8C91B96a31f0af8DD"
+# Address of the E2E test faucet
+E2E_FAUCET_ADDRESS="0xaF9053bB6c4346381C77C2FeD279B17ABAfCDf4d"
```

Comment:
Can we not use the .env in the root directory?

Reply:
Are you running the package from the root directory: like `cargo run -p celestia-prover` (or whatever the package is called)?

Reply:
I run it from inside the package so now I see why I added the env here. I'll remove.